### PR TITLE
feat(formatters): add nufmt config

### PR DESCRIPTION
**Pull request by abhisheksingh0x558**

### CODE REVIEW

File: lua/conform/formatters/nufmt.lua [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/nushell/nufmt",
+    description = "The nushell formatter",
+  },
+  command = "nufmt",
```
Comment (stevearc) [inline lua/conform/formatters/nufmt.lua:7]: Do you need to pass `--stdin` to get it to format from stdin/out? The docs seem to indicate that it's required.

**Review by stevearc — CHANGES_REQUESTED** [review]
One question

File: lua/conform/formatters/nufmt.lua (hunk 1 above)
Comment (abhisheksingh0x558) [reply lua/conform/formatters/nufmt.lua:7]: Thanks for pointing out. `$FILENAME` is what is required though.

Comment (stevearc) [reply lua/conform/formatters/nufmt.lua:7]: I would expect you to also need to add `stdin = false`

**Review by stevearc — CHANGES_REQUESTED** [review]

File: lua/conform/formatters/nufmt.lua (hunk 1 above)
Comment (stevearc) [reply lua/conform/formatters/nufmt.lua:7]: What I mean is, I would expect that either you are setting the option `stdin = false` and passing `args = { "$FILENAME" }` to indicate to the process that it should format a file, OR that you set `stdin = true` (or leave it out as that is the default) and pass `args = { "--stdin" }`. What I suspect is happening right now is that conform thinks it is running a formatter with stdin/stdout, but `nufmt` is formatting a file. That will work in simple cases, but in some situations cause very subtle bugs that are hard to track down.

Comment (abhisheksingh0x558) [reply lua/conform/formatters/nufmt.lua:7]: Got it! You are referring to conform option.
Fixed it!

**Comment by stevearc** [comment]
Thanks for the PR!

**Pull request merged**